Makefile

```diff
@@ -224,3 +224,13 @@ setup-buildx: ## Setup a Buildx builder
 
 promscale.control: ## A hack to boostrap the build, some pgx commands require this file. It gets re-generated later.
 	cp templates/promscale.control ./promscale.control
+
+.PHONY: devcontainer
+devcontainer:
+	docker build -f dev.Dockerfile -t devcontainer .
+
+.PHONY: devenv
+devenv: devcontainer
+	docker volume inspect build-volume 1>/dev/null 2>&1 || docker volume create build-cache
+	docker run --rm -vbuild-cache:/tmp/target ubuntu bash -c "chmod a+w /tmp/target"
+	docker run -ti --rm -vbuild-cache:/tmp/target -p54321:28814 -p8432:8000 -v$(shell pwd):/code devcontainer
```

Comment:
I got the following messages from this container. Is that normal?

```
FATAL:  extension "timescaledb" must be preloaded
HINT:  Please preload the timescaledb library via shared_preload_libraries.

This can be done by editing the config file at: /home/ubuntu/.pgx/data-14/postgresql.conf
and adding 'timescaledb' to the list in the shared_preload_libraries config.
        # Modify postgresql.conf:
        shared_preload_libraries = 'timescaledb'

Another way to do this, if not preloading other libraries, is with the command:
        echo "shared_preload_libraries = 'timescaledb'" >> /home/ubuntu/.pgx/data-14/postgresql.conf

(Will require a database restart.)

If you REALLY know what you are doing and would like to load the library without preloading, you can disable this check with:
        SET timescaledb.allow_install_without_preload = 'on';
server closed the connection unexpectedly
        This probably means the server terminated abnormally
        before or while processing the request.
connection to server was lost
==> /home/ubuntu/.pgx/14.log <==
                shared_preload_libraries = 'timescaledb'

        Another way to do this, if not preloading other libraries, is with the command:
                echo "shared_preload_libraries = 'timescaledb'" >> /home/ubuntu/.pgx/data-14/postgresql.conf

        (Will require a database restart.)

        If you REALLY know what you are doing and would like to load the library without preloading, you can disable this check with:
                SET timescaledb.allow_install_without_preload = 'on';
2022-07-11 14:31:24.418 GMT [45] STATEMENT:  CREATE EXTENSION IF NOT EXISTS timescaledb

==> /home/ubuntu/compile.log <==
entr: broken inotify workaround enabled
Error:
   0: control file not found in current directory

Location:
   pgx-utils/src/lib.rs:557

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ SPANTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

   0: cargo_pgx::command::install::install_extension with pg_version=14.14 release=false features=[]
      at cargo-pgx/src/command/install.rs:53
   1: cargo_pgx::command::install::execute
      at cargo-pgx/src/command/install.rs:36

```

Reply:
Oh damn, this was a leftover of my testing. Will remove.

Reply:
Hmm. Unsurprisingly, I got the same error message, but it's two pages up and I didn't notice it 🙈